cfpb/cfgov-refresh
# JS Fixes for rate checker

JS Fixes for rate checker


## Changes

- Modified rate-checker JS to do the following:
  - removed the word `County` from the dropdown.
  - fix issue rendering timestamp.
  - modified `revert to our default values` link behavior to reload page. 

## Testing

1. Visit `http://localhost:8000/owning-a-home/explore-rates/reload-link`.
2. Verify that the timestamp ( `rates are current as of` ) is rendered.
2. Slide the range slider to 600.
3. Click the `revert to our default values` in the notification. 
4. Verify that the page reloads.
5. Enter `1,000,000` in the house price input. 
6. Verify that the word county is removed from the `County` dropdown.

## Screenshots


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
